src/policy/immix/immixspace.rs

```diff
@@ -436,7 +486,11 @@ impl<VM: VMBinding> ImmixSpace<VM> {
     ) -> ObjectReference {
         let copy_context = worker.get_copy_context_mut();
         debug_assert!(!super::BLOCK_ONLY);
-        let forwarding_status = ForwardingWord::attempt_to_forward::<VM>(object);
+        let forwarding_status = if Self::is_pinned(object) {
+            0
```

Comment:
Does `0` mean `FORWARDING_NOT_TRIGGERED_YET`?
https://github.com/mmtk/mmtk-core/blob/644628bcc39e649e4125c092ad325bddbd24aaaf/src/util/object_forwarding.rs#L9

If so, you should use the constant.

Reply:
Another question is that do you really need any change in this method at all?I think the old code should work fine if the object is pinned.

Reply:
I'll try it out and see if that's the case. I think I had this change when the pinning bit was "shared" with the forwarding bits, but you're probably right that this method doesn't really need to change.